app/views/users/index.html.erb

```diff
@@ -0,0 +1,47 @@
+<h4> Users List</h4>
+<strong>
+  <p class="red-text center">
+    <% if @users.count == 0 %>
+      No User Record!
+    <% end %>
+  </p>
+  <p class="green-text">
+    <% if flash[:success] %>
+        <%= flash[:success] %>
```

Comment:
I find this conditional statement confusing, it probably can be better refactored

Reply:
I have refactored this flash message into the application layout since it has a global scope on all the view files